e2e/ctl_v3_kv_test.go

```diff
@@ -145,6 +146,25 @@ func getRevTest(cx ctlCtx) {
 	}
 }
 
+func getKeysOnlyTest(cx ctlCtx) {
+	var (
+		kvs = []kv{{"key1", "val1"}}
+	)
+	for i := range kvs {
+		if err := ctlV3Put(cx, kvs[i].key, kvs[i].val, ""); err != nil {
+			cx.t.Fatalf("getKeysOnlyTest #%d: ctlV3Put error (%v)", i, err)
+		}
+	}
+
+	cmdArgs := append(cx.PrefixArgs(), "get")
+	cmdArgs = append(cmdArgs, []string{"--prefix", "key", "--keys-only"}...)
+
+	err := spawnWithExpects(cmdArgs, []string{"val1"}...)
```

Comment:
shall we expect []string{"key1", ""} and do not expect error instead? (key1 as key and empty as val?)